test/test_public_bindings.py

```diff
@@ -288,6 +288,21 @@ def onerror(modname):
         # do not get imported by public code.
         private_allowlist = {
             "torch._inductor.codegen.cuda.cuda_kernel",
+            "torch.onnx._internal.exporter._analysis"
+            "torch.onnx._internal.exporter._building"
+            "torch.onnx._internal.exporter._capture_strategies"
+            "torch.onnx._internal.exporter._core"
+            "torch.onnx._internal.exporter._decomp"
+            "torch.onnx._internal.exporter._dispatching"
+            "torch.onnx._internal.exporter._fx_passes"
+            "torch.onnx._internal.exporter._ir_passes"
+            "torch.onnx._internal.exporter._isolated"
+            "torch.onnx._internal.exporter._onnx_program"
+            "torch.onnx._internal.exporter._registration"
+            "torch.onnx._internal.exporter._reporting"
+            "torch.onnx._internal.exporter._schemas"
+            "torch.onnx._internal.exporter._tensors"
+            "torch.onnx._internal.exporter._verification"
```

Comment:
Looks like this is happening again...

Reply:
This is needed and it is internal so I think we should be fine